### PR TITLE
Fixes #20450 - Update pagination to patternfly style

### PR DESCRIPTION
**Pull request by rabajaj0509**

### CODE REVIEW

**Comment by lzap** [comment]
Nice can you update also discovery rules? That's all we have, works just fine.

**Comment by lzap** [comment]
[test]

File: app/controllers/discovered_hosts_controller.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def index
       :organization,
       :model,
       :discovery_attribute_set
-    ], {:interfaces => :subnet}).paginate(:page => params[:page])
+    ], {:interfaces => :subnet}).paginate(:page => params[:page], :per_page => params[:per_page])
```
Comment (tbrisker) [inline app/controllers/discovered_hosts_controller.rb]: please use `@hosts = resource_base_search_and_page.includes(....` so if something changes in core we won't have to update this as well.

**Review by tbrisker — CHANGES_REQUESTED** [review]

**Comment by rabajaj0509** [comment]
@tbrisker, works perfectly now.

Looks like:
screenshot from 2018-01-31 17-02-45

**Review by tbrisker — APPROVED** [review]
LGTM, Thanks @rahulbajaj0509 !

**Pull request merged**